aqua/LyX/Portfile

```diff
@@ -6,9 +6,62 @@ PortGroup           boost 1.0
 
 name                LyX
 conflicts           LyX1
-version             2.4.2
-revision            0
-set branch          [join [lrange [split ${version} .] 0 1] .]
+
+# 2.4.x has expermintal support for Qt6 - we are ignoring ATM
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    PortGroup qt4 1.0
+
+    version     2.3.8
+    revision    0
+    checksums           rmd160  80b2c53d12b04dd64a6326b4b1ea505dcb352120 \
+                        sha256  7e8f56d148cb459eb00abbe9c6371744e4ff077b842c0024153a3f094b10d4ad \
+                        size    16412700
+
+    set branch          [join [lrange [split ${version} .] 0 1] .]
+
+    configure.pre_args  --prefix=${applications_dir}/LyX.app
+    configure.args-append \
+                        --without-x \
+                        --without-included-boost \
+                        --enable-optimization=-Os \
+                        --with-version-suffix=-${branch} \
+                        --with-qt-dir=${prefix}/libexec/qt4 \
+                        --disable-qt5
+
+    # QRawFont is supposed to work, but does not:
+    # GuiFontMetrics.cpp:127:9: error: 'QRawFont' was not declared in this scope; did you mean 'QFont'?
+    # Anyway, it seems to have issues in Qt4:
+    # https://stackoverflow.com/questions/12597595/qrawfontfromfont-not-working-correctly
+    patchfiles-append   patch-GuiFontMetrics.cpp.diff
+} else {
+    PortGroup qt5 1.0
+
+    version             2.4.2
+    revision            0
+
+    checksums           rmd160  686630c5fe6d98bb29625e0818d9ad027bdb3814 \
+                        sha256  0d5a75dec9a169ab23bfa47da53ec65c62432de57014de315023d620ebccfe68 \
+                        size    17871168
+
+    qt5.depends_component       qtbase \
+                                qtmacextras \
+                                qtsvg
+
+    set branch          [join [lrange [split ${version} .] 0 1] .]
+
+    configure.cxxflags-append -std=c++11
```

Comment:
this implies you need a compiler that supports the C++11 standard, but further down you request C++14 compatible compiler; that is inconsistent.